inc/html.php

```diff
@@ -41,59 +41,20 @@ function html_wikilink($id,$name=null,$search=''){
  * @author   Andreas Gohr <andi@splitbrain.org>
  *
  * @param bool $svg Whether to show svg icons in the register and resendpwd links or not
+ * @deprecated 2020-07-18
```

Comment:
Is it useful to log deprecation messages to the debug log as well? Or are this to many messages? Maybe more a question for @splitbrain 

On the long term, the idea is to drop the html_*() functions?

Reply:
deprecated functions should contain a call to `dbg_deprecated()` - that will log the call. In master to the debug log, with #3230 to a dedicated deprecation log.

Reply:
added `dbg_deprecated()` calls in 87f229e